guides/rules/mixins.md

```diff
@@ -31,14 +31,15 @@ export default Mixin.create({
     });
   }
 });
+```
 
 This example assumes that there is a store object on the consuming class. It also is going to set some properties on the consuming class which is another implicit side-effect. Another thing to note is because we assemble the concrete object at runtime (merging the mixins), we have to be careful about naming collisions with the consuming class. All of these downsides make this code make harder to test then other ways of sharing code.
```

Comment:
Yea this needs work. I just wrote a lot of stuff I didn't want to throw away. There is no rule for the mixin stuff yet.